iphone/Classes/PlatformModule.m

```diff
@@ -271,8 +271,34 @@ - (NSNumber *)openURL:(NSArray *)args
   NSString *newUrlString = [args objectAtIndex:0];
   NSURL *newUrl = [TiUtils toURL:newUrlString proxy:self];
   BOOL result = NO;
+
+  // iOS 10+
+  KrollCallback *callback = nil;
+  NSMutableDictionary *options = [NSMutableDictionary dictionary];
+
+  if ([args count] > 0) {
+    if ([[args objectAtIndex:1] isKindOfClass:[NSDictionary class]]) {
```

Comment:
In case of count = 1, [args objectAtIndex:1] will fail. Due to this first test case is failing.